src/content/en/updates/2017/10/devtools-release-notes.md

```diff
@@ -53,14 +53,15 @@ of VS Code and DevTools in action, side-by-side.
 </div>
 
 Multi-client remote debugging was the [number 1 most-popular DevTools issue
-on Crbug][Crbug], and number 3 across the entire Chromium project.
+on crbug.com][Crbug], and number 3 across the entire Chromium project.
 Multi-client support also  opens up quite a few interesting opportunities
 for integrating other tools with DevTools, or using those tools in new
 ways. For example:
 
 [Crbug]: https://crbug.com/129539
 
-* Two WebSocket protocol clients, such as two Puppeteer sessions, can now
+* An open DevTools window plus a connected protocol client. This includes Selenium/Chromedriver, Chrome debugging within VS Code or WebStorm, and WebSocket clients like Puppeteer. 
```

Comment:
Writer nitpick: Can you rephrase this to something like:

`Protocol clients such as ChromeDriver or the Chrome debugging extensions for VS Code and Webstorm, and WebSocket clients such as Puppeteer, can now run at the same time as DevTools.`

The rest of the items in the list are complete sentences, so this one should be, too. See [parallel construction](http://www.bartleby.com/141/strunk5.html#15)